test/integration/targets/emr_cluster/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+shippable/aws/group2
+unstable
```

Comment:
New tests should not be marked unstable. If the tests are truly unstable they should be fixed before merging.

Reply:
ok I just fixed it now, but I saw the tests are failing because I can't create the default IAM roles used by EMR, https://app.shippable.com/github/ansible/ansible/runs/115793/104/console.

What I've got to do?